### ISSUE TITLE
module: add XEP-0045(Multi-User Chat) support

### ISSUE THREAD

**Issue by kyokuki** [ortuman/jackal#58]
Do you have the plan to implement the XEP-0045(Multi-User Chat) ？

**Comment by ortuman** [comment]
Yeah, definitely! I’m currently working on component interface, and most likely Http upload will be first the implemented one. But from there on, I’ll consider to implement MUC as well. :)

**Comment by mechmind** [comment]
Looking forward for this module so I can finally ditch personal ejabberd :)

Note: i'd suggest to keep this issue open until implementation is ready so people skimming through issue list wont be confused.

**Comment by mmalesev** [comment]
Resolved in the PR #120

**Comment by apinprastya** [comment]
@ortuman the PR is on hold since Nov 2020, please let me know when you have any plan to reopen again the PR in the near future.

**Comment by Neustradamus** [comment]
Any news about it?

**Comment by Neustradamus** [comment]
There was a problem here: Initial release
ortuman committed on 22 Feb 2021

All the code has been broken since the moment.
